strings-js/js/script.js

```diff
@@ -0,0 +1,11 @@
+var text = 'Velociraptor is a genus of herbivorous ceratopsid dinosaur that first appeared during the late Maastrichtian stage of the late Cretaceous period.';
+var dinosaur = 'Triceratops';
+var bigDinosaur = dinosaur.toUpperCase();
+var text = text.replace('Velociraptor',bigDinosaur);
```

Comment:
deklarowales juz zmienna `text` wiec jesli ja nadpisujesz, to nie trzeba juz `var`